### ISSUE TITLE
Relation Chat.belongsTo(User)

### ISSUE THREAD

**Issue by Shenrak** [ShinGecko/neKo-Api#7]
@justinrlle I have a bug generated by your code in model relations
```js
Chat.belongsTo(User, 'user', 'private.idUser1', 'id')
Chat.belongsTo(User, 'user', 'private.idUser2', 'id')
```
Gives me an error `Error: The field 'user' is already used by another relation.`
That is quite new as i created `index.js` in app/models.

I'm surprised that that code worked before actually. Any idea ?

**Comment by justinrlle** [comment]
To be frank... I didn't test that code. It was just to have the logic somewhere in the code.
Just change the code to:
```javascript
Chat.belongsTo(User, 'user1', 'private.idUser1', 'id')
Chat.belongsTo(User, 'user2', 'private.idUser2', 'id')
```
And it should work

**Comment by Shenrak** [comment]
What is exactly the purpose to that code ? Make private chats between two users ?

**Comment by justinrlle** [comment]
Yup, that's it. I tried to do some kind of subclassing straight from the model, but it might not be a good workaround...
